docs/data/base/pages.ts

```diff
@@ -1,12 +1,10 @@
 import type { MuiPage } from 'docs/src/MuiPage';
-import standardNavIcons from 'docs/src/modules/components/AppNavIcons';
```

Comment:
Maybe time to delete this `standardNavIcons`

It has been introduced to enforce consistency between projects. But the new consistency seems to be "no icon"

Reply:
True, we still use a few icons, but we might not need a standard 👍